### PR TITLE
Add classes

### PR DESCRIPTION
**Pull request by paulinagonzalezc**
## OOP school library: add Person, Student and Teacher classes 

In this project, I started building my school library app. 
In this initial step, I implemented the classes to represent students and teachers.✨

* Create class Person with the mentioned requirements
* Create class Student with the mentioned requirements
* Create class Teacher with the mentioned requirements

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  # accesor get and set method
+  attr_accessor :name, :age
+  # accesor get method
+  attr_reader :id
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  def of_age?
```
Comment (cynthiainga) [inline person.rb:17]: - [x] God job so far, but the method that you use before is `is_of_age` but you define `of_age`. It should be the same method, either you use `of_age` or `is_of_age` it's for you to choose. Kindly fix that 🙃 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @paulinagonzalezc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] All linters are passing ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Correct workflow ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (paulinagonzalezc) [reply person.rb:17]: Hello @cynthiainga , you're right! Let me fix that 😄

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @paulinagonzalezc , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Gitflow is used correctly :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**